ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,22 @@
+# EupsPkg config file. Sourced by 'eupspkg'
+
+TAP_PACKAGE=1
+CONFIGURE_OPTIONS="--prefix=$PREFIX"
+
+install()
+{
+    die_if_empty PRODUCT
+    die_if_empty VERSION
+
+    clean_old_install
+
+    # Headers and libs only
```

Comment:
This isn't strictly necessary given that the executable will only affect people if the table file enables it.